docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -405,6 +405,22 @@ Follow these steps:
    python3 main.py
    ```
 
+## Using MIOpen kdb files with ROCm PyTorch wheels
+
+PyTorch uses MIOpen for machine learning primitives. These primitives are compiled into kernels at runtime. Runtime compilation causes a small warm-up phase when starting PyTorch. MIOpen kdb files contain precompiled kernels that can speed up the warm-up phase of an application. More information is available in the [MIOpen documentation](inv:miopen:install#installing-miopen-kernels-package).
```

Comment:
The link does not seem to render correctly: https://advanced-micro-devices-demo--2248.com.readthedocs.build/en/2248/how_to/pytorch_install/pytorch_install.html#using-miopen-kdb-files-with-rocm-pytorch-wheels

The `reference/all.md` page uses this syntax:
```
{doc}`MIOpen <miopen:index>`
```

Perhaps this would work:
```
{doc}`MIOpen <miopen:install#installing-miopen-kernels-package>`
```

Reply:
There is no roc-5.6.x branch on MIOpen. I think will need wait for @Maetveis to add support for the release candidate branches. That is causing your current syntax from working.